spec.bs

```diff
@@ -309,6 +342,10 @@ The <dfn attribute for=HTMLFencedFrameElement>config</dfn> IDL attribute getter
   1. <span class=XXX>TODO</span>
 </div>
 
+The IDL attribute
+<dfn attribute for="fencedframe"><code data-x="dom-fencedframe-allow">allow</code></dfn>
+</dfn> must [=reflect=] the respective content attributes of the same name.
```

Comment:
In this PR there is no content attribute. You'll need to add an attribute to this section: https://github.com/WICG/fenced-frame/blob/master/spec.bs#L229